docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,24 @@
+= Identifying Policy in-use
+
+In Konflux, there are two places where Enterprise Contract Policies are used, integration tests and
+release pipelines.
+
+
+== Integration Tests
+
+For integration tests, the Policy is defined in an IntegrationTestScenario resource. These are
+stored in the same namespace as your Application. They are created automatically when a new
+Application is created via the Web UI.
+
+IntegrationTestScenarios have multiple purposes. Running EC validation is just one of them. As such,
+you may have multiple of these resources in your namespace. The one that runs EC points to
+`pipelines/enterprise-contract.yaml` Tekton Pipeline in the
+`https://github.com/konflux-ci/build-definitions` repository.
+
+Update, or set, the value of the `POLICY_CONFIGURATION` parameter to your new Policy.
+
+== Release
+
+The release pipeline uses EC as a gating step. The `.spec.policy` attribute of the
+ReleasePlanAdmission associated with your Application specifies which Policy is used during a
```

Comment:
Would it make more sense to include this information in with the RPA section? Or at least a call-out in the RPA for specifying an ECP, xref'ing here for more information?

Reply:
Added a link to it.